IPython/html/static/notebook/js/kernelselector.js

```diff
@@ -78,7 +78,7 @@ define([
             new_notebook_submenu.append(
                 $("<li>").attr("id", "new-notebook-submenu-"+ks.name).append(
                     $('<a>')
-                        .attr('href', '#')
+                        .attr('href', 'javascript:void(0)')
```

Comment:
It is common but not very good in the long run. I think it should change, but I agree it doesn't need to change in this PR.